docs/dev-environment-setup.md

```diff
@@ -0,0 +1,155 @@
+# Getting Your Dev Environment Started
+
+This guide will provide you with the information you need to be able to get started on your development journey. This document will include:
+
+- Which text editors can you install?
+- What Tools/plugins you need to install?
+- Installing Node.js, Node Version Manager (NVM), NPM, NPX, and Git
+
+### Contents
+
+[Text Editors](#Text-Editors)
+
+[Useful Extensions to Install](#Useful-Extensions-to-Install)
+
+[Installing Node.js, NVM, NPM, NPX, and Git ](#Installing-Node.js,-NVM,-NPM,-NPX,-and-Git)
```

Comment:
This skip-link is invalid.

Reply:
You are indeed correct.  I'll get that fixed.